src/common-components/Logistration.jsx

```diff
@@ -101,14 +110,29 @@ Logistration.propTypes = {
   intl: intlShape.isRequired,
   selectedPage: PropTypes.string,
   backupRegistrationForm: PropTypes.func.isRequired,
+  tpaProvidersAndSecondaryProviders: PropTypes.shape({
+    providers: PropTypes.array,
+    secondaryProviders: PropTypes.array,
+  }),
+};
+
+Logistration.defaultProps = {
+  tpaProvidersAndSecondaryProviders: {
+    providers: [],
+    secondaryProviders: [],
+  },
 };
 
 Logistration.defaultProps = {
   selectedPage: REGISTER_PAGE,
 };
 
+const mapStateToProps = state => ({
+  tpaProvidersAndSecondaryProviders: tpaProvidersAndSecondaryProvidersSelector(state),
```

Comment:
Lets change the name for this. TPA stands for third party auth. We can just name is as follows:
```suggestion
  tpaProviders: tpaProvidersSelector(state),
```